SimpleITK/utilities/fft.py

```diff
@@ -0,0 +1,60 @@
+import SimpleITK as sitk
+
+
+def fft_based_translation_initialization(fixed: sitk.Image, moving: sitk.Image):
```

Comment:
Add type hint for return type,`-> sitk.TranslationTransform`? I don't use type hints much so may be mistaken here.